fr-batch-service/docs/runbook.md

```
@@ -0,0 +1,350 @@
+# Operational Runbook — fr-batch-service
+
+## 1. System Overview
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)